core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/flexsubscriptions/responses/FlexSubscriptionV4Response.java

```diff
@@ -16,11 +16,8 @@
     @ApiModelProperty(value = ModelDescriptions.ID, readOnly = true)
     private Long id;
 
-    @ApiModelProperty(value = ModelDescriptions.PUBLIC_IN_ACCOUNT, readOnly = true)
-    private boolean publicInAccount = true;
-
     @ApiModelProperty(value = FlexSubscriptionModelDescription.SMARTSENSE_SUBSCRIPTION, readOnly = true)
```

Comment:
What does it mean `readOnly`

Reply:
to be honest, i do not know, maybe the property cannot be changed after parsing the json, internet says the `readOnly` will be automatically `true` if we are not creating setter for the property